src/sentry/tasks/commit_context.py

```diff
@@ -48,6 +48,8 @@
 PR_COMMENT_TASK_TTL = timedelta(minutes=5).total_seconds()
 PR_COMMENT_WINDOW = 14  # days
 
+PR_COMMENT_SUPPORTED_PROVIDERS = {"integrations:github"}
```

Comment:
when all SCMs have implemented the `get_merge_commit_sha_from_commit` function, we can remove this